libs/blocks/fragment/fragment.js

```diff
@@ -2,6 +2,7 @@
 import { createTag, getConfig, loadArea, localizeLink, customFetch } from '../../utils/utils.js';
 
 const fragMap = {};
+window.fm = fragMap;
```

Comment:
I'd assume this can be removed, I didn't see it used in this file, nor did I see it in unit tests.

Reply:
Yep leftover debug code.  Thanks!